src/main.c

```diff
@@ -51,7 +51,7 @@ void get_device_info(void)
         printf("  Target: %s\n", target);
 
     is_mac = !!strstr(model, "Mac");
-    has_dcp = adt_path_offset(adt, "/arm-io/dcp") > 0;
+    has_dcp = !!adt_path_offset(adt, "/arm-io/dcp") || !!adt_path_offset(adt, "/arm-io/dcpext0");
```

Comment:
unrelated to the fix. this should use the display_config from display.c. I also don't see the need to handle this outside display.c. The called functions should just return immediately when running on device without dcp. I can fix that tonight